pybossa/repositories/task_repository.py

```diff
@@ -79,18 +79,20 @@ def filter_completed_taskruns_gold_taskruns_by(self, limit=None, offset=0,
         last_id=None, yielded=False, desc=False, **filters):
 
         exp = filters.pop('exported', None)
+        finish_time = filters.pop('finish_time', None)
         filters.pop('state', None) # exclude state param
-        if exp is not None:
-            query = self.db.session.query(TaskRun).join(Task).\
-                filter(TaskRun.task_id == Task.id).\
-                filter(or_(Task.state == u'completed', Task.calibration == 1)).\
-                filter(Task.exported == exp).\
-                filter_by(**filters)
-        else:
-            query = self.db.session.query(TaskRun).join(Task).\
-                filter(TaskRun.task_id == Task.id).\
-                filter(or_(Task.state == u'completed', Task.calibration == 1)).\
-                filter_by(**filters)
+
+        conditions = []
+        if exp:
+            conditions.append(Task.exported == exp)
+        if finish_time:
+            conditions.append(TaskRun.finish_time >= finish_time)
+
+        query = self.db.session.query(TaskRun).join(Task).\
```

Comment:
If you enclose an expression in `()` you can break across lines without the `\`. See https://github.com/bloomberg/pybossa/blob/15c88b1f2fad83843dd1895e247d7abdee4faccf/pybossa/repositories/task_repository.py#L149-L152